### PR TITLE
feat:  Add example for deploy apigatewayv2 websocket lambda application

### PR DESCRIPTION
**Pull request by ypwu1**
feat: Add example of  using apigatewayv2 and lambda, dynamodb.  

---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

**Comment by ypwu1** [comment]
Could somebody have a look ?

**Comment by ypwu1** [comment]
Doesn't seem to have any progress ... 😞

**Comment by kewur** [comment]
bump

**Comment by cstst** [comment]
bump

File: typescript/api-websocket-lambda-dynamodb/index.ts [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+                "dynamodb:UpdateItem",
+                "dynamodb:BatchWriteItem",
+                "dynamodb:BatchGetItem",
+                "dynamodb:DescribeTable",
+                "dynamodb:ConditionCheckItem"
+            ],
+            resources: [table.tableArn]
+        });
+
+        const connectLambdaRole = new Role(this, "connectLambdaRole", {
```
Comment (omakoleg) [inline typescript/api-websocket-lambda-dynamodb/index.ts:51]: `connectLambdaRole` is used only in one lambda. Maybe will be reasonable just to link them with `grant` like:
`connectFunc.grantReadWrite(table.tableArn)`

Comment (ypwu1) [reply typescript/api-websocket-lambda-dynamodb/index.ts]: I have updated it.

**Comment by ypwu1** [comment]
Bump and review, please.

**Review by ryparker — APPROVED** [review]
Successfully deployed to AWS account

**Pull request merged**